src/wallet/wallet2.cpp

```diff
@@ -1194,6 +1194,16 @@ void wallet2::set_seed_language(const std::string &language)
   seed_language = language;
 }
 //----------------------------------------------------------------------------------------------------
+const std::time_t &wallet2::get_creation_time() const
```

Comment:
If you can get and set creation time, there's no need for getter and setter functions. Just make the variable public